src/gcodeExport.cpp

```diff
@@ -649,16 +677,19 @@ void GCodeExport::writeTravel(int x, int y, int z, double speed)
 
     const PrintFeatureType travel_move_type = extruder_attr[current_extruder].retraction_e_amount_current ? PrintFeatureType::MoveRetraction : PrintFeatureType::MoveCombing;
     const int display_width = extruder_attr[current_extruder].retraction_e_amount_current ? MM2INT(0.2) : MM2INT(0.1);
-    CommandSocket::sendLineTo(travel_move_type, Point(x, y), display_width, layer_height, speed);
+    const double layer_height = Application::getInstance().current_slice->scene.settings.get<double>("layer_height");
```

Comment:
Fixed: 1e35013de3104497cc7ce29f7e589c29778204e8

I also looked through other uses of "layer_height" but these were the only two places where they were obtained globally. I misremembered that they were global settings rather than per-meshgroup.